### PR TITLE
Add Sign and Verify transit methods

### PR DESCRIPTION
**Pull request by vejed**

### CODE REVIEW

File: transit_test.go [hunk 1]
```diff
@@ -238,3 +238,73 @@ func (s *TransitTestSuite) TestCreateKeyThatDoesAlreadyExist() {
 	err = s.client.Create("testCeateKeyThatDoesAlreadyExist", &TransitCreateOptions{})
 	require.NoError(s.T(), err)
 }
+
+func (s *TransitTestSuite) TestImplicitSignCreate() {
+	_, err := s.client.Sign("test404", &TransitSignOptions{
```
Comment (Lucaber) [inline transit_test.go:243]: Try to use a different key name like `testImplicitSignCreate` to ensure a new key is generated

File: transit.go [hunk 2]
```diff
@@ -308,6 +310,165 @@ func (t *Transit) DecryptBatch(key string, opts TransitDecryptOptionsBatch) (*Tr
...
+	err := t.client.Write([]string{"v1", t.MountPoint, "verify", url.PathEscape(key)}, opts, res, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	return res, nil
+}
+
+// DecodeCipherText gets payload from vault ciphertext format (removes "vault:v<ver>:" prefix)
+func DecodeCipherText(vaultCipherText string) (string, error) {
```
Comment (Lucaber) [inline transit.go:457]: Return the `keyVersion` too. This allows us to re-encode a ciphertext with `EncodeCipherText` from the information returned by `DecodeCipherText`

**Review by Lucaber — CHANGES_REQUESTED** [review]

File: go.mod [hunk 3]
```diff
@@ -1,4 +1,4 @@
-module github.com/mittwald/vaultgo
+module github.com/vejed/vaultgo
```
Comment (Lucaber) [inline go.mod:1]:
```suggestion
module github.com/mittwald/vaultgo
```

Comment (vejed) [reply go.mod:1]: I chooe invalid source branch, sorry.
Please, close this PR and view another one (other conversations are also fixed) https://github.com/mittwald/vaultgo/pull/21

**Pull request closed without merging**